interface/src/app/download/download.js

```diff
@@ -1,80 +1,74 @@
 /**
- * "download" module, which will download sample annotations of given
- * annotation typenames.
+ * "adage.download" module, which will download sample annotations based on
+ * selected annotation types.
  */
 
 angular.module('adage.download', [
   'ui.router',
-  'placeholders',
   'ui.bootstrap',
+  'ngResource',
```

Comment:
This is just a side comment - I agree that we should include these dependencies on each module that requires them. I was thinking about it, and even though ``ngResource`` is included in the 'adage' module itself in ``app.js``, and could technically be omitted here, I agree that we should explicitly include it here since every one of these modules should be a stand-alone packet of code.